src/components/Chat/Chat.vue

```diff
@@ -228,17 +228,30 @@
       </template>
 
       <template #fab>
-        <v-btn
-          v-if="!isScrolledToBottom"
-          class="ma-0 grey--text"
-          color="grey lighten-3"
-          depressed
-          fab
-          size="small"
-          @click="$refs.chat.scrollToBottom()"
-        >
-          <v-icon icon="mdi-chevron-down" size="x-large" />
-        </v-btn>
+        <div>
+          <v-badge
+            :modelValue="numOfNewMessages > 0 ? true : false"
+            floating
+            location="top center"
+            v-if="!isScrolledToBottom"
```

Comment:
I think you can remove `v-if` on `v-badge` component, because it is already used by `v-btn` below.